src/components/routing/Navigation/styles.css

```diff
@@ -3,5 +3,27 @@
 }
 
 .navigation {
-  bottom: 0;
+  position: relative;
+  height: 80px;
+}
+
+.navigation-link {
+  background-color: #e1dada;
+  border-radius: 50%;
+  height: 60px;
+  width: 60px;
+  border: none;
+  margin: 0 15px;
+  color: #000;
+  text-decoration: none;
+
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  font-size: 25px;
+}
+
+.selected {
+  background-color: #d9af62;
+  color: white;
```

Comment:
Only idea (don't require changes if you don't want to do it):

What do you think about using the same color for buttons or links (in this case the same yellow tone) 🟡 ? 🤔 

![image](https://user-images.githubusercontent.com/8689897/142735690-6534b13a-0c79-423d-934e-4855b1e8b28b.png)